### PR TITLE
Restrict form counts query to only selected users

### PR DESCRIPTION
**Pull request by esoergel**
Current behavior pulls stats on all users even when filtered.  This is only relevant when sorting by a column other than username (the default).  Just something I stumbled across randomly.

@emord FYI - I think you've been looking in to report optimizations, wanted to flag in case you had any reservations.
@millerdev @mkangia buddies

### CODE REVIEW

**Review by emord — APPROVED** [review]
Been a while since I looked at general reports. This seems fine to me. nice find.

Might be nice to change the property all_users to selected_users, but I don't know how tricky that is (like if it is inheriting that from another class)

**Comment by esoergel** [comment]
@emord good call.  It looked pretty straightforward, so I just did it.

File: corehq/apps/reports/standard/monitoring.py [hunk 1]
```diff
@@ -53,17 +53,16 @@
...
 from dimagi.utils.decorators.memoized import memoized
 from dimagi.utils.parsing import json_format_date, string_to_utc_datetime
-from django.utils.translation import ugettext as _, ugettext_lazy
-from django.utils.translation import ugettext_noop
+
 import six
 from functools import reduce
 from six.moves import range
 from six.moves import map
+from six.moves.urllib.parse import urlencode
```
Comment (emord) [inline corehq/apps/reports/standard/monitoring.py]: Init  think pep8 is formatted as

```
std library
third party libs
your app
```

So I think functools should be at the top and six next to django right?

**Review by emord — APPROVED** [review]

File: corehq/apps/reports/standard/monitoring.py (hunk 1 above)
Comment (esoergel) [reply corehq/apps/reports/standard/monitoring.py]: I was thinking of this as part of a 2-to-3 compatibility block, although I guess it's technically a stdlib import.  IDK, no strong feelings on it.

**Pull request merged**